### PR TITLE
Add package.json

### PR DESCRIPTION
**Pull request by josh**
To: @notwaldorf

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "url-polyfill",
+  "version": "0.5.6",
```
Comment (josh) [inline package.json]: What version are we at?

Comment (notwaldorf) [inline package.json]: I just tagged 0.5.7

**Comment by notwaldorf** [comment]
👀 @azakus WDYT?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "url-polyfill",
```
Comment (josh) [inline package.json]: Not sure what the package name should be or if you intend to actually publish this. I'd be fine referencing it at its git url or whatever. `URL` is definitely taken already on pm.

**Comment by azakus** [comment]
LGTM

**Comment by notwaldorf** [comment]
Oooooh merginnnn

**Pull request merged**